wlauto/utils/fps.py

```diff
@@ -12,17 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import collections
+
+try:
+    import pandas as pd
+except ImportError:
+    pd = None
 
 
 class FpsProcessor(object):
+    SurfaceFlingerFrame = collections.namedtuple('SurfaceFlingerFrame', 'desired_present_time actual_present_time frame_ready_time')
+    GfxInfoFrame = collections.namedtuple('GfxInfoFrame', 'Flags IntendedVsync Vsync OldestInputEvent NewestInputEvent HandleInputStart AnimationStart PerformTraversalsStart DrawStart SyncQueued SyncStart IssueDrawCommandsStart SwapBuffers FrameCompleted')
+    GfxInfoSummary = collections.OrderedDict((('Total frames rendered', '0'),
+                                              ('Janky frames', '0 (nan%)'),
+                                              ('90th percentile', '133ms'),
```

Comment:
After performing a reset, these values are the 'default' values that gfxinfo gives you.
I am mimicking the defaults.